layouter/cms_plugins.py

```diff
@@ -19,6 +19,12 @@ class ContainerCMSPlugin(CMSPluginBase):
         (None, {
             'fields': ('container_type', 'margin')
         }),
+        (_('Background'), {
```

Comment:
I like it. I'm thinking about organizing the admin in tabs, not in accordions in the future - this could make the overview a bit more clear. But not for now.